tldw_Server_API/app/core/Agent_Client_Protocol/adapters/mcp_transport.py

```
@@ -0,0 +1,98 @@
+        MCPStreamableHTTPTransport,
+    )
+
+    protocol = protocol_config.get("mcp_transport", "")
+
+    if protocol == "stdio":
+        return MCPStdioTransport(
+            command=protocol_config["command"],
+            args=protocol_config.get("args"),
+            env=protocol_config.get("env"),
+        )
+    elif protocol == "sse":
```

Ruff (0.15.6)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)